frontend/src/main/components/BasicCourseSearch/CourseOverTimeBuildingsSearchForm.js

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+
+import { allBuildings } from "fixtures/buildingFixtures";
+import { quarterRange } from "main/utils/quarterUtilities";
+
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+import SingleBuildingDropdown from "../Buildings/SingleBuildingDropdown";
+
+const CourseOverTimeBuildingsSearchForm = ({ fetchJSON }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker enable OptionalChaining
+
+  const quarters = quarterRange(startQtr, endQtr);
+
+  // Stryker disable all : not sure how to test/mock local storage
+  const localStartQuarter = localStorage.getItem("CourseOverTimeBuildingsSearch.StartQuarter");
+  const localEndQuarter = localStorage.getItem("CourseOverTimeBuildingsSearch.EndQuarter");
+  const localBuildingCode = localStorage.getItem("CourseOverTimeBuildingsSearch.BuildingCode")
+
+  const [startQuarter, setStartQuarter] = useState(localStartQuarter || quarters[0].yyyyq);
+  const [endQuarter, setEndQuarter] = useState(localEndQuarter || quarters[0].yyyyq);
+  const [buildingCode, setBuildingCode] = useState(localBuildingCode || {});
+    
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    fetchJSON(event, { startQuarter, endQuarter, buildingCode });
+  };
+
+  // Stryker disable all : Stryker is testing by changing the padding to 0. But this is simply a visual optimization as it makes it look better
```

Comment:
This line should be removed according to the professor's comments on #42. You'll probably need a new test to handle the padding style.